test/test_repolib.py

```diff
@@ -853,6 +859,149 @@ def test_releasever_the_string_none(self):
         self.assertTrue(YumReleaseverSource.is_set({'releaseVer': 'None'}))
 
 
+if HAS_DEB822:
```

Comment:
Is there any reason why not run following unit test on other platform? If unit tests are written correctly, then it should not be problem to run them on other platforms.

Reply:
Because the class, which is tested, is only defined if the library can be imported (compare [repofile](https://github.com/candlepin/subscription-manager/blob/7ebf96eaf1edd53efa5f2ea0efdb082ef2d66e95/src/subscription_manager/repofile.py#L393)) .

Reply:
Sure, but you can mock the `HAS_DEB822` in `repofile.py`. Something like `@mock.patch(repofile.HAS_DEB822, return_value=True)`. If these unit tests were platform specific, then we could break the functionality easily, because we run unit tests only on fedora machine on Jenkins.

Reply:
Alright, thanks for the advice!

Reply:
I am not sure what's the best way to mock a import from an external module. So I just changed `sys.modules` which worked fine for me. What do you think about it?